blockboard/src/main/resources/static/js/event/common.js

```diff
@@ -26,4 +26,37 @@ function errorFunction(xhr) {
     alert(jsonResponse.message);
     redirectLogout();
   }
+}
+
+function getByteLength(size,maxLength,byte,i,ch){
```

Comment:
byte, i 가 들어와서 다시 초기화 되네요.
지역 변수가 더 좋을것 같아요

```suggestion
function getByteLength(size,maxLength){
  var byte, i, ch;
```